spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/condition/OnResourceCondition.java

```diff
@@ -47,7 +47,7 @@ public ConditionOutcome getMatchOutcome(ConditionContext context,
 					? this.defaultResourceLoader : context.getResourceLoader();
 			List<String> locations = new ArrayList<String>();
 			collectValues(locations, attributes.get("resources"));
-			Assert.isTrue(locations.size() > 0,
+			Assert.isFalse(locations.isEmpty(),
```

Comment:
This doesn't compile as there's no `isFalse` method on `org.springframework.util.Assert`.